force-app/main/default/lwc/carouselImage/__tests__/carouselImage.spec.js

```diff
@@ -7,9 +7,9 @@
 
 import { createElement } from 'lwc';
 import Element from 'c/carouselImage';
-import { axe, toHaveNoViolations } from 'jest-axe';
+import { registerSa11yMatcher } from '@sa11y/jest';
```

Comment:
@sgesin Does this actually work?

Reply:
@vbhargavmbis The unit tests pass locally and lwc-recipes started including sa11y too:
https://github.com/trailheadapps/lwc-recipes/blob/master/jest-sa11y-setup.js